CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
-# CHANGELOG.md
+# Changelog
+
+All the notable changes to this project will be documented in this file.
+
+## v1.1.0 (2021-01-05)
+### Features:
+- [standardize navbar](https://github.com/Real-Dev-Squad/website-my/issues/171)
+- [add loading state for all routes](https://github.com/Real-Dev-Squad/website-my/issues/163)
```

Comment:
@swarajpure you mentioned this issue over here, but didn't mention it in features below😅.
Or maybe add it under UX section.

Reply:
@akshay1502 These are the features which we included in v1.1.0, and below are features which were there in v1.0.0. and from this version, Swaraj starts adding issues for the respective features.
From now on this will be format. (Linking respective issues with feature)

Reply:
Yes, that's correct understanding @rohan09-raj. Thanks for the explanation!
Also, if you check conventional commits (which ankush is planning to start for all repos soon), there is no seperate category for UX. It has got features, fixes, documentation etc. Can check it from here: https://gist.github.com/qoomon/5dfcdf8eec66a051ecd85625518cfd13